<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>251</width>
      <height>351</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="widget_2" native="true">
       <widget class="QPushButton" name="btn_SelectFile">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Select File</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_SelectedFile">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>231</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_SaveDir">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>50</y>
          <width>151</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Select Saving Directory</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_SaveDir">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>80</y>
          <width>231</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <widget class="QLineEdit" name="lineEdit_From">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_From">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>56</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string># From</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_To">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>20</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_To">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>0</y>
          <width>56</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string># To</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_3" native="true">
       <widget class="QPushButton" name="btn_MakePDFs">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Make PDFs</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>251</width>
          <height>51</height>
         </rect>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBar">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>90</y>
          <width>241</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_File"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
